### ISSUE TITLE
Documentation on non-JS initiatlization

### ISSUE THREAD

**Issue by basilgohar** [year-calendar/js-year-calendar#41]
The Getting Started guide shows we can initialize a bare calendar without any JS calls by using:
```html
<div data-provide="calendar"></div>
```
But I did not find in the documentation any further explanation of other ways to initialize a calendar – for example, can we pre-populate a calendar with events/days using additional data attributes and are there any other date-* attributes that can be used to bootstrap a calendar quickly without having to use extensive JS?

If such functionality exists, it would be great to have it be documented and/or easier to discover.

**Comment by basilgohar** [comment]
Scanning through the code for examples where `data` attributes are utilized, I did not find any additional examples beyond the one above, so I take it there's not much more here.  I guess if you want more than an empty table it's best to just initialize it like a regular Javascript widget.  I'll close this issue out myself now.

**Comment by Paul-DS** [comment]
Indeed, there is no other option available with HTML initialization.
I rename and reopen this issue so people can indicate if they are interested to have more options available with HTML initialization.
